packages/react-components/src/components/HighContrastAwareIcon.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Icon, mergeStyles } from '@fluentui/react';
+import React from 'react';
+import { useTheme } from '../theming';
+
+/**
+ * Strings of HighContrastAwareIcon that can be overridden.
```

Comment:
nit: This comment isn't quite true. The `props` include more than strings. The strings comment is more useful for publicly exported components.